core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -336,12 +336,41 @@ var ExplorationEditorMainTab = function() {
         var ruleDescription = _getRuleDescription(interactionId, ruleName);
         // Replace selectors with feedbackTextArray's elements.
         ruleDescription = _replaceRuleInputPlaceholders(
-          ruleDescription, feedbackTextArray);
+          ruleDescription, feedbackTextArray, interactionId);
         ruleDescription += '...';
         // Adding "..." to end of string.
         var answerTab = element(by.css('.protractor-test-answer-tab'));
         expect(answerTab.getText()).toEqual(ruleDescription);
       },
+      /**
+       * Check for correct rule parameters in the case of multiple rules in an
+         answer group.
+       * @param {string[][]} - A 2 dimensional array , where each 1 dimensional
+         array contains the parameters for a particular rule.
+       */
+      expectMultipleRulesToBe: function(ruleParamatersArray) {
+        element.all(by.css('.protractor-test-answer-tab')).then(
+          function(answerTabs) {
+            for (var i = 0; i < ruleParamatersArray.length; i++) {
+              var ruleDescription = _getRuleDescription(
+                ruleParamatersArray[i][0], ruleParamatersArray[i][1]);
+                // From second rule onwards, we need add an "or" before the
+                // rule description.
+              if (i > 0) {
+                ruleDescription = 'or ' + _replaceRuleInputPlaceholders(
+                  ruleDescription, ruleParamatersArray[i][2],
+                  ruleParamatersArray[i][0]);
+              } else {
+                ruleDescription = _replaceRuleInputPlaceholders(
+                  ruleDescription, ruleParamatersArray[i][2],
+                  ruleParamatersArray[i][0]);
+              }
+              ruleDescription += '...';
+              // Adding "..." to end of string.
```

Comment:
I think people generally expect comments to reference subsequent, not previous, lines

```suggestion
              // Adding "..." to end of string.
              ruleDescription += '...';
```

Reply:
done